app/views/catalog/_facet_layout.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="panel panel-default facet_limit blacklight-<%= facet_field.key.parameterize %> <%= 'facet_limit-active' if facet_field_in_params?(facet_field) %>">
+  <div tabindex="0" role="button" class="panel-heading facet-field-heading collapse-toggle <%= "collapsed" if should_collapse_facet?(facet_field, display_facet) %>" data-toggle="collapse" data-target="#<%= facet_field_id(facet_field) %>" aria-expanded="<%= !should_collapse_facet?(facet_field, display_facet) %>">
+    <span class="sr-only">Filter search results by</span>
+    <span class="panel-title"><%= facet_field_label(facet_field.key) %></span>
```

Comment:
@gkostin1966 was there a specific reason why the facet panel title was changed from a `<h3>`  to a `<span>`?

On preview:
<img width="402" alt="Screen Shot 2019-11-11 at 2 44 20 PM" src="https://user-images.githubusercontent.com/1686111/68615832-d182ca00-0491-11ea-8e23-a7a2652d6714.png">

On production:
<img width="381" alt="Screen Shot 2019-11-11 at 2 44 35 PM" src="https://user-images.githubusercontent.com/1686111/68615883-e6f7f400-0491-11ea-8e69-942323a3b07f.png">

Reply:
It may have been when I was wrestling with GA.  I switch h3 for span and made a commit so it can easily be rollback.